### PR TITLE
BT2H-4 Implement Temp Sensor and LCD screen

### PR DESCRIPTION
**Pull request by ammonson3**
DESCRIPTION: added temp sensor and lcd screen to arduino code

TESTS: hardware tests

### CODE REVIEW

File: Hardware/Temp___Lcd.ino [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+// Data wire is plugged into digital pin 2 on the Arduino
+#define ONE_WIRE_BUS 2
+
+// Setup a oneWire instance to communicate with any OneWire device
+OneWire oneWire(ONE_WIRE_BUS);  
+
+// Pass oneWire reference to DallasTemperature library
+DallasTemperature sensors(&oneWire);
+/**********************************************************/
+char array1[]=" SunFounder               ";  //the string to print on the LCD
```
Comment (ykalra) [inline Hardware/Temp___Lcd.ino]: array1 and array 2 could be deleted as they are unused in the code

File: Hardware/Temp___Lcd.ino [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+char array2[]="hello, world!             ";  //the string to print on the LCD
+int tim = 250;  //the value of delay time
```
Comment (ykalra) [inline Hardware/Temp___Lcd.ino]: tim could also be deleted, if unused

**Review by ykalra — CHANGES_REQUESTED** [review]
Apart from these small things, the code looks great! Thank you!

**Review by riyabhat2002 — APPROVED** [review]
Everything looks good! Thanks

**Review by riyabhat2002 — CHANGES_REQUESTED** [review]
I tried making this circuit and the temperature sensor works but it doesn't show anything on the LCD screen. Could you put up the circuit that you used initially in which the LCD screen showed the temperature?

**Pull request merged**